poetro_app/lib/features/poetry/presentation/widgets/poetry_previw_item.dart

```diff
@@ -1,25 +1,25 @@
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
-import 'package:poetro_app/config/routes/routes.dart';
+import 'package:poetro_app/config/routes/app_routes.dart';
 import 'package:poetro_app/core/const/app_colors.dart';
 import 'package:poetro_app/features/poetry/presentation/models/poetry_model.dart';
 
 class PoetryPreviwItem extends StatelessWidget {
-  final PoetryModel poetryDTO;
+  final PoetryModel poetryModel;
```

Comment:
PoetryModel i think is exptra you can us Poetry entity instead it Models are for any logics like validation and you not keep somthing like it in your model and not need it in presentation layer